samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -230,26 +222,51 @@ export const ChatInput: React.FC<ChatInputProps> = ({ isDraggingOver, onDragLeav
                 />
             </div>
             <div className={classes.controls}>
-                {!features[FeatureKeys.SimplifiedExperience].enabled && <div className={classes.functional}>
-                    {/* Hidden input for file upload. Only accept .txt and .pdf files for now. */}
-                    <input
-                        type="file"
-                        ref={documentFileRef}
-                        style={{ display: 'none' }}
-                        accept=".txt,.pdf,.md,.jpg,.jpeg,.png,.tif,.tiff"
-                        multiple={true}
-                        onChange={() => {
-                            handleImport();
-                        }}
-                    />
-                    <Button
-                        disabled={documentImporting}
-                        appearance="transparent"
-                        icon={<AttachRegular />}
-                        onClick={() => documentFileRef.current?.click()}
-                    />
-                    {documentImporting && <Spinner size="tiny" />}
-                </div>}
+                {!features[FeatureKeys.SimplifiedExperience].enabled && (
+                    <div className={classes.functional}>
+                        {/* Hidden input for file upload. Only accept .txt and .pdf files for now. */}
+                        <input
+                            type="file"
+                            ref={documentFileRef}
+                            style={{ display: 'none' }}
+                            accept=".txt,.pdf,.md,.jpg,.jpeg,.png,.tif,.tiff"
+                            multiple={true}
+                            onChange={() => {
+                                void fileHandler.handleImport(
+                                    setDocumentImporting,
+                                    documentFileRef,
+                                    undefined,
+                                    imageUploaderRef,
+                                );
+                            }}
+                        />
+                        <Button
+                            disabled={documentImporting}
+                            appearance="transparent"
+                            icon={<AttachRegular />}
+                            onClick={() => documentFileRef.current?.click()}
+                        />
+                        <FileUploader
```

Comment:
Why do we need a separate button just for images? The one we have already supports images.

Reply:
Removed - treating images as docs